extension/js/common/core/crypto/smime/smime-key.ts

```diff
@@ -1,18 +1,23 @@
 /* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
-import * as forge from 'node-forge';
 import { Key, UnexpectedKeyTypeError } from '../key.js';
 import { Str } from '../../common.js';
 import { UnreportableError } from '../../../platform/catch.js';
 import { PgpArmor } from '../pgp/pgp-armor.js';
 import { Buf } from '../../buf.js';
 import { MsgBlockParser } from '../../msg-block-parser.js';
 import { MsgBlock } from '../../msg-block.js';
-
+import * as forge from 'node-forge';
 export type SmimeMsg = forge.pkcs7.PkcsEnvelopedData;
 export const DATA_OID = '1.2.840.113549.1.7.1';
 export const SIGNED_DATA_OID = '1.2.840.113549.1.7.2';
 export const ENVELOPED_DATA_OID = '1.2.840.113549.1.7.3';
 export class SmimeKey {
+  private static get forgeCustom(): typeof forge {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-expect-error
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+    return forge.default ? forge.default : forge;
+  }
```

Comment:
`forgeCustom` sounds like our custom implementation, what about renaming to `forgeDefault` or `forgeLib`?